shell/tools/copy-external-code.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2021 Terminus, Inc.
```

Comment:
Suggest move this file to enterprise and rename the script as `npm run extra-logic`, so the logic represented as a black box.